openapi-cli/src/main/java/io/ballerina/openapi/cmd/utils/CodegenUtils.java

```diff
@@ -36,6 +39,22 @@ public static Path getImplPath(String pkg, Path srcPath) {
         return (pkg == null || pkg.isEmpty()) ? srcPath : srcPath.getParent();
     }
 
+    /**
+     * Resolves path to write generated implementation source files.
+     *
+     * @param fileName     name of the file which contains the content of license header
+     */
+    public static void setLicenseHeaderFilePath (String fileName) throws IOException {
+        licenseHeader = "";
+        if (!fileName.isBlank()) {
+            Path licenseHeaderPath = Paths.get((new File(fileName).getCanonicalPath()));
+            licenseHeader = Files.readString(Paths.get(licenseHeaderPath.toString()));
+            if (!licenseHeader.endsWith("\n")) {
```

Comment:
There is a scenario if the user mistakenly appends a new line to the end of content, then it won't append a new line at the end. Do we need to consider it? :)

Reply:
Added extra if check here. Please review